ios/Podfile

```diff
@@ -9,11 +9,11 @@ end
 
 target 'Plugin' do
   capacitor_pods
-  pod 'NewRelicAgent', '7.4.8'
+  pod 'NewRelicAgent', '7.4.10'
```

Comment:
required because we refer to `NRMAFeatureFlags.NRFeatureFlag_OfflineStorage` which AFAIK is not part of 7.4.8. I just used the most recent version of the agent, not sure if 7.4.9 is ok and preferred?